### DISCUSSION TITLE
snyk test support (esp. Go projects)

### DISCUSSION THREAD

**Discussion by mcandre** [community/community#14098] [Code Security] [answered]
Snyk now offers support for scanning Go projects. dependabot would be improved by running `snyk test` on Go repositories and emailing the usual CVE reports for any findings.

**Comment by jhutchings1** [comment, marked as answer]
Dependabot already supports Go projects natively. Thanks! https://docs.github.com/en/enterprise-cloud@latest/get-started/learning-about-github/github-language-support
